app/models/organization.rb

```diff
@@ -21,6 +22,8 @@ class Organization < ApplicationRecord
   after_update_commit :conditionally_update_articles
   after_destroy_commit :bust_cache
 
+  pg_search_scope :search_organizations, against: :name
```

Comment:
Would it be useful to search against the slug as well? That would match the search behaviour for users